tcms/core/tests/test_views.py

```diff
@@ -28,15 +28,26 @@ def setUpTestData(cls):
         # used to reproduce Sentry #KIWI-TCMS-38 where rendering fails
         # with that particular value
         cls.chinese_tp = TestPlanFactory(name="缺货反馈测试需求", author=cls.tester)
-        doc_url = "https://kiwitcms.readthedocs.io/en/latest/admin.html#configure-kiwi-s-base-url"
+        doc_url_base = (
+            "https://kiwitcms.readthedocs.io/en/latest/admin.html"
+            "#configure-kiwi-s-base-url"
+        )
         cls.base_url_error_message = _(
             "Base URL is not configured! "
             'See <a href="%(doc_url)s">documentation</a> and '
             '<a href="%(admin_url)s">change it</a>'
         ) % {
-            "doc_url": doc_url,
+            "doc_url": doc_url_base,
             "admin_url": reverse("admin:sites_site_change", args=[settings.SITE_ID]),
         }
+        doc_url_ssl = (
+            "https://kiwitcms.readthedocs.io/en/latest/installing_docker.html"
+            "#ssl-configuration"
+        )
+        cls.ssl_error_message = _(
+            "You are not using secure connection (SSL). "
```

Comment:
Change to "You are not using a secure connection. See documentation and enable SSL."